### DISCUSSION TITLE
Unity 2022.3.5f1

### DISCUSSION THREAD

**Discussion by vengefullfoot** [codemagic-ci-cd/codemagic-docs#1915] [Q&A] [answered]
### Use case

```Text
When will have access to Unity 2022.3 stream (the latest LTS) ? This version is mandatory for a some critical features of our apps...
```

### Idea

```Text
Add Unity 2022.3 to Mac vms.
```

**Comment by icarusdust** [comment]
Hi @vengefullfoot, I assuming that you mean `2022.3.0` because there is .1, .2, .3, .4. Please confirm it, so that I can proceed with this request.

**Comment by vengefullfoot** [comment]
Dear Nihal,

Thank you for taking in account my request so quickly. My heart was broken that I cannot use code magic for my project…
I was referring to the LTS stream 2022.3 as the version in the stream is less a blocker. Now if you want a full version number, 2022.3.5f1 would be the best for me.
Also, will you make this version of unity available on some of the new vmd like the future M2 ?

Best regards,
Jerome



Le 27 juil. 2023 à 08:59, Nihal Agazade ***@***.***> a écrit :

because there is .1, .2, .3, .4. Please confirm it, so that I can proceed with this request.

**Reply by mikhail-tokarev** [reply, marked as answer]
Hi @vengefullfoot 
We have added `2022.3.5f1` version. Can you give it another try? Thank you.

**Comment by vengefullfoot** [comment]
It's working fine now.

Thank you very much for the swift update of the VM !
